### PR TITLE
README and LICENSE cleanups

### PR DESCRIPTION
**Pull request by sashko**

### CODE REVIEW

**Comment by sashko** [comment]
Would like a feedback on `README: organize dependencies in a table` -- not sure if this actually looks better.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-META-PELUX
-==========
+# meta-pelux
+
```
Comment (DunderRoffe) [inline README.md]: I much prefer the preset style, using ==== dividers over #
Makes it easier to read in a text editor :)

File: README.md [hunk 2]
```diff
@@ -11,25 +11,26 @@ layer are specific for the PELUX image but can be used outside of this context.
 
 Maintained at https://github.com/pelagicore/meta-pelux
 
-Dependencies
-------------
+## Dependencies
```
Comment (DunderRoffe) [inline README.md]: See above

File: README.md [hunk 3]
```diff
@@ -11,25 +11,26 @@ layer are specific for the PELUX image but can be used outside of this context.
...
 The relationship with `meta-bistro`, which is also developed at Pelagicore,
 is that `meta-bistro` contains general software recipes and configurations,
 aimed at the IVI domain, but no distribution and no images, whereas meta-pelux
 builds on top of meta-bistro by adding a distro and recipes to build complete
 system images.
 
-Branching
----------
+## Branching
+
```
Comment (DunderRoffe) [inline README.md]: See above

File: README.md [hunk 4]
```diff
@@ -40,8 +41,8 @@ Security and bug fixes will be evaluated case by case and backported as
 necessary. The ambition is to actively maintain the two latest releases and/or
 one year old releases in this fashion.
 
-License and Copyright
----------------------
+## License and Copyright
```
Comment (DunderRoffe) [inline README.md]: See above

**Comment by DunderRoffe** [comment]
@sashko: feedback I think the update README: organize dependencies in a table looks better :)

**Comment by sashko** [comment]
@DunderRoffe it would make it inconsistent between our projects :(

**Comment by thenor** [comment]
@sashko On the other hand this would destroy the consistency within PELUX. Is there a reason why the other project you want to align with can't change instead?

**Comment by sashko** [comment]
@thenor I have never seen any project writing it as META-QT, META-OPENEMBEDDED, etc. it's always meta-xyz

**Comment by thenor** [comment]
@sashko Now I understand what you meant with the capital letters, but this is not the formatting I meant. I actually don't mind having "meta-pelux", what I didn't like was the formatting for headers ('#' instead of '=').

**Comment by sashko** [comment]
@thenor yeah, it's a very minor issue not worth arguing - whatever you guys prefer.

**Pull request closed without merging**